prboom2/src/p_spec.c

```diff
@@ -6099,25 +6099,25 @@ dboolean P_ExecuteZDoomLineSpecial(int special, int * args, line_t * line, int s
       buttonSuccess = EV_DoZDoomCeiling(ceilLowerByValue, line, args[0],
                                         P_ArgToSpeed(args[1]), 0,
                                         args[2], P_ArgToCrush(args[4]), 0,
-                                        P_ArgToChange(args[3]), false);
+                                        P_ArgToChange(args[3]), crushDoom);
```

Comment:
`false` and `crushDoom` are very different things, even if they have the same result, so this is going to need a more substantial solution. I'll take a look at this one myself, so you can drop it from this PR.

Reply:
`EV_DoZDoomCeiling` explicitly takes a `crushmode_e`.  I dropped these changes from the PR.

Reply:
I went into the zdoom source and it uses `crushDoom` here as well, so `false` is actually a mistake in the first place. Maybe this predates the enum. If you could open a PR adding the changes back, I would appreciate it 😸